src/autoyast-rnc/kdump.rnc

```diff
@@ -41,10 +41,8 @@ kdump_general = element general {
     element KDUMP_REQUIRED_PROGRAMS { STRING }? &
     element KDUMP_PRESCRIPT { STRING }? &
     element KDUMP_POSTSCRIPT { STRING }? &
-    element KDUMPTOOL_FLAGS { STRING }? &
     element KDUMP_NETCONFIG { STRING }? &
     element KDUMP_NET_TIMEOUT { STRING }? &
-    element KDUMP_COPY_KERNEL { STRING }? &
```

Comment:
this param is quite documented in autoyast documentation. So removal should follow also PR to sle documentation https://github.com/SUSE/doc-sle/blob/main/xml/ay_kernel_dumps.xml#L310